src/components/Admin/Toolbar.jsx

```diff
@@ -4,9 +4,34 @@ import Checkbox from "../Checkbox";
 import { HiSearch } from "react-icons/hi";
 import Tag from "./Tag.jsx";
 import { FaDownload, FaTrashAlt } from "react-icons/fa";
+import { CSVLink } from "react-csv";
 import { colors } from "@/data/Tags";
 
-const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
+function convert(input) {
+  if (Array.isArray(input)) {
+    return input.map(convert).join(", ");
+  } else if (typeof input === "boolean") {
+    return input ? "True" : "False";
+  } else if (typeof input === "object") {
+    return Object.entries(input)
```

Comment:
can u show an example of this mapping and join, like what does it look like?

Reply:
it turns {a: 1, b: 2} to "a: 1, b: 2"

Reply:
- can we just do 1, 2, 3...
- does the generated CSV have a table header?

Reply:
- I don't know about 1,2,3...
- There is no table header prop that you can pass that I know of. We would need to do some hard coding of the "data" prop, so then we get a "header" in the CSV file

Reply:
headers is getting passed into the table component and the toolbar component, so we should be able to add the headers as the first row